package-lock.json

```diff
@@ -0,0 +1,5384 @@
+{
```

Comment:
we don't want to use package-lock.json files at this point.  the partial support in the world right now keeps leading to broken installs.  for now, let's stick to tighter versioning in package.json.

Reply:
I've removed it.

Reply:
Thanks